apertium-kaa.kaa.lexc

```diff
@@ -26459,6 +26464,7 @@ sonsha:sonsha N1 ;
 son:son N1 ;
 soń:soń N1 ;
 són:són V-IV ;
+soń:soń POST-NOM ;
```

Comment:
Also sometimes POST-ABL?  E.g., barǵan soń, but also mektepten soń?  (I'm making up these examples and they could be wrong, so let's discuss.)

Reply:
yesó you are right. Examples are right.

Reply:
@IlnarSelimcan, should we add to both POST-NOM and POST-ABL?  It shouldn't make a difference in terms of how it works, but I don't know your reasoning for dividing these to start with.